docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc

```diff
@@ -34,7 +34,7 @@ If they are blocked, you cannot enable the Cloud Services.
 
 The OpenNMS Cloud Services Connector is an extension that lets your {page-component-title} instance send time series data to the OpenNMS-hosted cloud service.
 The packages for this extension are installed by default in Horizon 31 and later, but they do need to be configured and activated.
-You can use `yum` or `apt` to make sure your `opennms-cloud-plugin` package is up to date.
+You can use `yum` or `apt` to make sure your `opennms-plugin-cloud` package is up-to-date.
```

Comment:
```suggestion
You can use `yum` or `apt` to make sure your `opennms-plugin-cloud` package is up to date.
```
This is one of those ones I always have to look up, but in this context, up to date is not hyphenated.